docs/components/camera/transform.md

```diff
@@ -140,6 +142,8 @@ Classifications overlay text from the `GetClassifications` method of the [vision
 
 - `classifier_name`: The name of the classifier in the [vision service](/ml/vision/).
 - `confidence_threshold`: The threshold above which to display classifications.
+- `max_classifications`: The maximum number of classifications to display on the camera stream at any given time.
+- `valid_labels`: An array of labels that you to see detections for on the camera stream.
```

Comment:
Do we know for sure that these are all required? Would be good to point out if not.

Reply:
I don't think they are but will check

Reply:
These are both optional. They have "under the hood" defaults that may or may not be worth sharing. If you don't put anything for max_classifications, it'll just put one classification (which is what we've always had and what most expect). If you don't put anything for valid_labels, it'll display all the labels your classifier returns (again, expected and always had).